### DISCUSSION TITLE
Positions in the result are 0-based?

### DISCUSSION THREAD

**Discussion by loganylchen** [GoekeLab/xpore#75] [Q&A] [answered]
Hi there,

May I ask if the position in the final result was 0-based?

Best,

**Comment by ploy-np** [comment, marked as answer]
Hi @btrspg,
The final result table reported in [xpore.readthedocs.io](https://xpore.readthedocs.io/en/latest/quickstart.html) and our paper is genomic one-based coordinates. However, when you run xpore, the results will depend on what kind of index system used in the annotation file (gtf) you provide.

**Comment by AminaLEM** [comment]
Hi @ploy-np , 

If I run xpore on transcriptomic mode, does it consider 0-based annotation? I guess that from the middle position in the Kmer column ...

Thanks!
